test/verify/check-machines-dbus

```diff
@@ -806,14 +808,12 @@ class TestMachinesDBus(machineslib.TestMachines):
         # Remove all Virtual Networks and confirm that trying to choose
         # Virtual Networks type for a NIC disables the save button
         m.execute("virsh net-dumpxml default > /tmp/net-default.xml")
-        self.addCleanup(m.execute, "rm /tmp/net-default.xml")
         m.execute("virsh net-dumpxml test_network > /tmp/net-test-network.xml")
-        self.addCleanup(m.execute, "rm /tmp/net-test-network.xml")
         m.execute("virsh net-destroy test_network && virsh net-destroy default")
-        self.addCleanup(m.execute, "virsh net-start default")
+        self.addCleanup(m.execute, "virsh net-define /tmp/net-default.xml && virsh net-autostart default")
```

Comment:
This seems to be something that the MachineCase could do centrally? If it's the only test that touches the default network, it's fine to keep here, of course.